xchange-stream-coinjar/src/main/java/info/bitrich/xchangestream/coinjar/CoinjarStreamingExchange.java

```diff
@@ -0,0 +1,70 @@
+package info.bitrich.xchangestream.coinjar;
+
+import info.bitrich.xchangestream.core.ProductSubscription;
+import info.bitrich.xchangestream.core.StreamingExchange;
+import info.bitrich.xchangestream.core.StreamingTradeService;
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+import org.knowm.xchange.coinjar.CoinjarExchange;
+
+public class CoinjarStreamingExchange extends CoinjarExchange implements StreamingExchange {
+
+  private static final String API_URI = "wss://feed.exchange.coinjar.com/socket/websocket";
+
+  private CoinjarStreamingService streamingService;
+  private CoinjarStreamingMarketDataService streamingMarketDataService;
+  private CoinjarStreamingTradeService streamingTradeService;
+
+  @Override
+  protected void initServices() {
+    super.initServices();
+
+    this.streamingService = createStreamingService();
+    this.streamingMarketDataService = new CoinjarStreamingMarketDataService(streamingService);
+    this.streamingTradeService = new CoinjarStreamingTradeService(streamingService);
+  }
+
+  private CoinjarStreamingService createStreamingService() {
+    return new CoinjarStreamingService(API_URI, this.exchangeSpecification.getApiKey());
+  }
```

Comment:
being pedantic here but can you just inline this?

Reply:
Done